things/postgres/things_test.go

```diff
@@ -197,8 +127,8 @@ func TestThingUpdate(t *testing.T) {
 		Key:   thkey,
 	}
 
-	id, _ := thingRepo.Save(context.Background(), thing)
-	thing.ID = id
+	sths, _ := thingRepo.BulkSave(context.Background(), []things.Thing{thing})
```

Comment:
If every `Save()` i `BulkSave()` now, i.e. if there I would prefer that we name this `Save()` and know that Save() actually saves an array of things.

Reply:
That sounds good to me